picking_good_slis.md

```diff
@@ -0,0 +1,92 @@
+# Picking Good SLIs
+
+## SRE's Philosophy on Monitoring
+
+A core goal of SRE is to ensure the availability of a service. "Availability"
+here means much more than simply "is my application running." Effectively
+monitoring a modern cloud application requires insight into the application's
+response time, error rate, traffic level, and more.
+
+Furthermore, the scope of monitoring should not be limited solely for use in
+deciding when to page an operations team. Metrics should be closely tied to
+business objectives, informing decisions on what features to prioritize and
+when to focus on adding new features versus improving reliability.
+
+To make this vision a reality, SRE teams rely on two key concepts:
+  * Service Level Indicators (SLIs) - A quantifiable metric that measures a
+    specific aspect of a service's availability.
```

Comment:
```suggestion
    specific aspect of a service's level of service.
```

The definition from the [Google SRE book](https://sre.google/sre-book/service-level-objectives/) is 

> An SLI is a service level indicator—a carefully defined quantitative measure of some aspect of the level of service that is provided.

Avoiding the word availability is key here in my opinion which has been well explained in the first paragraph of this  document.